contracts/token/LivepeerToken.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.25;
+pragma solidity ^0.5.11;
```

Comment:
> is it safe to update the compiler version for already deployed non-upgradeable contracts?

Yes in that we're just using the contracts locally. But, the downside is that the test scenarios are not as "real" because on mainnet an upgraded contract, whether or not it is using a new compiler version or not, will be interacting with a contract compiled with an old compiler version. More on this later in the comment.

> I assume this changes the bytecode

Yes it should especially if the optimizer is turned on (we do turn it on).

> Are we going to redeploy these then?

No - I think we can get away with only deploying a new `BondingManager` and `RoundsManager` (in additional to the `TicketBroker`). Some changes were made to the `Minter` awhile ago, but I think those can be reverted - will write up in a separate GH issue.

> If not I would prefer to keep the compiler version here at 0.4.25 but change the interface for these contracts to the new version according to this guide if we want a higher version contract to interact with an older version contract.

Truffle doesn't natively support multiple compiler versions in the same project, but I like this idea so I'm going to investigate if there is a workaround today (maybe via the external compiler config).

An ideal scenario would be that the contracts in the repo will correspond exactly with contracts that will live on-chain after the upgrade. So, there will be some upgraded contracts with higher compiler versions and some old contracts with older compiler versions.

Will report back soon!